### PR TITLE
Added accessibility tests to notebook components

### PR DESCRIPTION
**Pull request by esanzgar**
Added a11y tests to these components:

- NotebookView
- FilterSelect
- NotebookResultCount
- PaginationNavigation

To following components I didn't add a11y tests because they are
wrappers of one or several components already tested:

- PaginatedThreadList (ThreadList + PaginationNavigation)
- NotebookFilters (FilterSelect)

Closes: #2869

### CODE REVIEW

File: src/sidebar/components/test/FilterSelect-test.js [hunk 1]
```diff
@@ -88,4 +92,27 @@ describe('FilterSelect', () => {
     assert.isFalse(wrapper.find('MenuItem').at(1).props().isSelected);
     assert.isTrue(wrapper.find('MenuItem').at(2).props().isSelected);
   });
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility([
+      {
+        content: () => {
+          $imports.$restore();
```
Comment (esanzgar) [inline src/sidebar/components/test/FilterSelect-test.js:101]: This line has the effect of disable `mock-imported-components` making the a11y more realistic.

File: src/sidebar/components/test/PaginationNavigation-test.js [hunk 2]
```diff
@@ -148,4 +150,11 @@ describe('PaginationNavigation', () => {
       });
     });
   });
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility({
+      content: () => createComponent({ currentPage: 2 }),
```
Comment (LMS007) [inline src/sidebar/components/test/PaginationNavigation-test.js:157]: 👍

**Review by LMS007 — APPROVED** [review]
I recommend just a few changes to test names and there seems to be an extra `<div>` I think we can remove. g2g otherwise.

File: src/sidebar/components/test/FilterSelect-test.js [hunk 3]
```diff
@@ -3,19 +3,23 @@ import { mount } from 'enzyme';
...
   const createComponent = props => {
     return mount(
-      <FilterSelect
-        defaultOption={{ value: '', display: 'all' }}
-        onSelect={() => null}
-        options={someOptions}
-        title="Select one"
-        {...props}
-      />
+      <div>
```
[the remark this answers is not in the document]
Comment (esanzgar) [reply src/sidebar/components/test/FilterSelect-test.js:14]: Yes, you are right.

File: src/sidebar/components/test/NotebookResultCount-test.js [hunk 4]
```diff
@@ -132,4 +134,36 @@ describe('NotebookResultCount', () => {
...
+
+  it(
+    'should pass a11y checks',
+    checkAccessibility([
+      {
+        name: 'no results',
+        content: () => createComponent({ isFiltered: true }),
+      },
+      {
+        name: 'no applied filter',
```
[the remark this answers is not in the document]
Comment (esanzgar) [reply src/sidebar/components/test/NotebookResultCount-test.js:146]: 👍🏼

**Pull request merged**